doc/backend_vars.asciidoc

```diff
@@ -7,6 +7,7 @@ Supported variables per backend
 |====================
 Variable;Values allowed;Default value;Explanation
 EXCLUDE_MODULES;string;;comma separated names or fullnames of excluded test modules
+EXIT_AFTER_SCHEDULE;boolean;0;Exit test execution immediately after evaluation of the test schedule, e.g. to check only which test modules would be executed
```

Comment:
as this is pretty internal, I would welcome if this was prefixed with _ - otherwise you will have someone who has a get_var('EXIT_AFTER_SCHEDULE') in her main.pm to test a random app and will be very confused.

Reply:
updated.

Doesn't this also apply for other variables? I find it hard to distinguish between internal and external variables. I guess this is also why you stated "pretty internal". I guess there is also "not so much internal" or "ugly internal" ;-)

Reply:
I thought about it more and I don't like it. I still updated the code to convince you that the PR is mergeable but I suggest to go back to the version without the underscore. What about SKIPTO? Shouldn't it be also prepended with leading underscore? What about QEMU_OVERRIDE_VIDEO_DEVICE_AARCH64? Will it not also have a potential big impact? The good thing about EXIT_AFTER_SCHEDULE is that the effect is drastic, people will learn faster than with a hideous change of a backend device setting. Maybe I should just add a diag message on exit to make it more explicit? I did that, updated (still with leading underscore). A diag message "Early exit has been requested with _EXIT_AFTER_SCHEDULE. Only evaluating test schedule." will be printed.

Reply:
SKIPTO is very old - and for new variables I e.g. insist on a QEMU_ prefix

But there is a difference between a variable that belongs in a machine and one that developers use to debug something in their main.pm